lib/Gateway.js

```diff
@@ -807,7 +812,8 @@ Gateway.prototype.parsePayload = function (payload, valueId, valueConf) {
         const parsedVal = evalFunction(
           valueConf.receiveFunction,
           valueId,
-          payload
+          payload,
+          {}
```

Comment:
I suggest to fetch the node here someway :) maybe add the node to the parsePayload function signature too (remember to update all calls of this function then

Reply:
@robertsLando  I guess I'll need some help here on how to proceed:
Have I seen it correctly that the `nodeId` can (always?) be accessed using `valueId.nodeId` and the node then using `this.zwave.nodes[nodeID]`?
If so, we do not even need to pass `node` to `evalFunction` at all since we could fetch `node` this way within that anyway.
Or am I missing something?

PS: I'm looking forward to have TypeScript here, as it always shows you the types without looking at every line that might put properties into an object to see the full list of properties ;-)

Reply:
Hmmm ... it seems not to work in `evalFunction` but from within `parsePayload` I can get the node using `this.zwave.nodes`.
It's already pushed, I'm not sure, if it is a safe approach though.

Reply:
> ave I seen it correctly that the nodeId can (always?) be accessed using valueId.nodeId and the node then using this.zwave.nodes[valueId.nodeId]?

Reply:
Hmm, it has been working for me so far.
Are there any edge cases that you can imagine?
Can you suggest a better way?

Reply:
@ahochsteger the way you did it is ok :)

Reply:
@ahochsteger Sorry maybe I explained wrong, it was ok to get the node in that way, instead of passing `{}`. Could you revert your last commit?

Reply:
Oh, sorry, it's reverted again.